packages/components/stencil.config.js

```diff
@@ -21,5 +22,9 @@ exports.config = {
         path.join(__dirname, '../styles/src/core/variables/_variables.scss')
       ]
     })
-  ]
+  ],
```

Comment:
Because to avoid weird spacing it has to be like this:
```
plugins: [
    sass({
      outFile: path.join(__dirname, '.generated-styles'),
      injectGlobalPaths: [
        path.join(__dirname, '../styles/src/core/variables/_variables.scss')
      ]
    })
  ],
  preamble: preamble: `Airship Components · CARTO · https://carto.com
Version ${version}`
};
```

and I didn't like it much :(